### ISSUE TITLE
Remove bot comment when stale label is removed

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [actions/stale#906]
**Description:**
I would like the bot to autoremove or hide its comment when the stale label has bwwn removed.

**Justification:**
Lets take https://github.com/FreeTubeApp/FreeTube/issues/2331 as an example. The label got removed several times but the bot comments are still there. In my opinion  it clutters the conversation allot.

**Are you willing to submit a PR?**

**Comment by MaksimZhukov** [comment]
Hello @efb4f5ff-1298-471a-8973-3d47447115dc! Thank you for the suggested idea!
We will consider adding this feature and will let you know as soon as we have any decision.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @MaksimZhukov i would like to come back to this issue. It would be nice if the comment could be hidden but i think its even better that it would delete the previous comment. 

Reason for this is even when comments get hidden the user still gets to scroll quite a bit (test for yourself in the issue i linked) and the user doesnt know if there is something usefull in the comments or not so they are forced to scroll to the bottom. Deleting the comments would prevent this. Also u will still see full stale history of the issue without all the comments because the issue also updates when a label gets removed or applied.

**Comment by fregante** [comment]
I agree that comments should be deleted after the label has been removed. The information is no longer useful and it's already part of the event history:

<img width="400" alt="Image" src="https://github.com/user-attachments/assets/a48827fb-6325-4e73-bb14-590f7a5dc982" />


No further information is required here.

**Comment by aattalla** [comment]
@MaksimZhukov any updates on this to be released soon? It'd be so helpful to be able to delete/hide the stale comments once the PR is active again. 🙏
